fe/fe-core/src/main/java/com/starrocks/connector/hive/HiveMetaStoreThriftClient.java

```diff
@@ -596,7 +618,16 @@ public Table getTable(String dbName, String tableName) throws MetaException, TEx
 
     @Override
     public Table getTable(String catName, String dbName, String tableName) throws MetaException, TException {
-        return client.get_table(dbName, tableName);
+        try {
+            // Using get_table_req() first, if user is using Hive v1.x version, it will request failed,
+            // then fail over to use get_table() instead.
+            GetTableRequest req = new GetTableRequest(dbName, tableName);
+            req.setCapabilities(version);
+            return client.get_table_req(req).getTable();
```

Comment:
why not use `get_table` at first

Reply:
Most people are using Hive version `>= 2.x`, which already supports `get_table_req()`.

Reply:
use get_table first +1